### ISSUE TITLE
Contents overlapping: Timeline with layers

### ISSUE THREAD

**Issue by narcisgarcia** [LibreTime/libretime#1222]
**Is your feature request related to a problem? Please describe.**
Calendar managing tools don't allow to overlap contents (shows).

**Describe the solution you'd like**
Allow to schedule overlapped contents in calendar, with a layers logical. Top layer at a time to be the only to play: Similar logical as non-linear video editor (picture).

**Describe alternatives you've considered**
I didn't consider ani alternative.

**Additional context**
Radio station may need to schedule music background for all hours and all days, and this feature can make easier to schedule simple tracks or blocks for a minute without needing to cut and move other scheduled contents: Station will be playing usual contents and on that time switches to scheduled audio for a minute and returns to lower content layer after.

**Comment by jooola** [comment]
I like the idea, this should definitely be considered once the migration to the new API  is done.

**Comment by paddatrapper** [comment]
I agree. This would make it easy to schedule adverts, show content and backup music as three separate layers

**Comment by narcisgarcia** [comment]
There should be 2 underlapping modes:
A) Run time silently while a superior layer plays
B) Pause while a superior layer plays, and resume once superior layer completes.

**Comment by narcisgarcia** [comment]
This feature request is not a critical matter for immediately work, but it has a high importance to enhance usability of LibreTime software.
Thank you for marking this issue as **_pinned_**.
